src/common/dbus.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "memory.h"
+
+#include <systemd/sd-bus.h>
+#include <systemd/sd-event.h>
+
+#define _cleanup_sd_event_ _cleanup_(sd_event_unrefp)
+#define _cleanup_sd_event_source_ _cleanup_(sd_event_source_unrefp)
+#define _cleanup_sd_bus_ _cleanup_(sd_bus_unrefp)
+#define _cleanup_sd_bus_slot_ _cleanup_(sd_bus_slot_unrefp)
+#define _cleanup_sd_bus_message_ _cleanup_(sd_bus_message_unrefp)
```

Comment:
What is the purpose of the leading underscores here?

Reply:
Using `_` for naming is called `snake case` - the [systemd codestyle](https://github.com/systemd/systemd/blob/main/docs/CODING_STYLE.md#code-organization-and-semantics) uses this (second bullet point).

Reply:
Right, should've made my question more clear - snake case I understand, I specifically meant the preffix/suffix underscores. I've read some heated flame wars about using/not using `_foo_` naming

Reply:
Don't we also need to bring in this line from orch.h?
`#define _cleanup_(x) __attribute__((__cleanup__(x)))`

Reply:
Ah, got it, thanks, @sdunnagan !